fluent-bit/coralogix/README.md

```diff
@@ -8,18 +8,49 @@ The default values we provide can be overriden according to your needs, the defa
 helm show values coralogix-charts-virtual/fluent-bit-coralogix
 ```
 
-## Installation
+## Installation with default/dynamic app_name and sub_system
+Dynamic App_Name and Sub_System means taking the value from the running containers, so it can be 'namespace', 'pod_name', 'container_name'.
+If you need to override the default values, but still use dynamic app_name and sub_name, please follow these installation instructions: 
 The following environment variables can be overriden via the 'set' flag in the upgrade command:
 * app_name
 * sub_system
 * endpoint
 * logLevel
 
 for example:
+```bash
+helm upgrade fluent-bit-coralogix coralogix-charts-virtual/fluent-bit-coralogix \
+  --install \
+  --namespace=<your-namespace> \
+  --create-namespace \
+  --set "fluent-bit.logLevel=<level>" \
+  --set "fluent-bit.app_name=<app_name>" \
+  --set "fluent-bit.sub_system=<sub_system>" \
+  --set "fluent-bit.endpoint=<Coralogix-endpoint>"
 ```
-helm upgrade fluent-bit-coralogix coralogix-charts-virtual/fluent-bit-coralogix --install --namespace=<your-namespace> --create-namespace --set "fluent-bit.logLevel=<level>"
---set "fluent-bit.app_name=<app_name>" --set "fluent-bit.sub_system=<sub_system>" --set "fluent-bit.endpoint=<Coralogix-endpoint>"
+
+## Installation with static app_name and sub_system
+Static App_Name and Sub_System means using hardcoded values, like 'production', 'test'. 
+If you need to override the default values, and use hardcoded values, then you need to create the following 'override.yaml' file instead of using '--set'.
+
+```yaml
+---
+#override.yaml
+fluent-bit:
+  config:
+    outputs: |-
+      [OUTPUT]
+          Name          coralogix
+          Endpoint      <Coralogix_endpoint>
+          Match         kube.*
+          Private_Key   ${PRIVATE_KEY}
+          App_Name      <new_app_name>
+          Sub_Name      <new_sub_system_name>
+
+      @INCLUDE output-systemd.conf
 ```
+** If you don't need to override both endpoint, app_name and sub_name, then keep the default configuration wherever you want to keep the default, for example, 
+put `${ENDPOINT}` instead of <Coralogix_endpoint> in order to keep the default value.
```

Comment:
We should make a note that our suggestion is to make it dynamic, which is more agile way